nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -254,11 +277,11 @@ in
             echo "file sd-image $img" >> $out/nix-support/hydra-build-products
           fi
 
-          root_fs=${rootfsImage}
+          root_fs=${config.sdImage.rootImage}
```

Comment:
```suggestion
          root_fs=${config.sdImage.rootFilesystemImage}
```

Reply:
Sorry it is late and two other projects keep me busy. I thought I tested for that, but I just quickly launched a `nixos-rebuild` which builds `config.system.build.toplevel` but these changes only effect `config.system.build.sdImage`.